views/job-role-list.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+
+{% block title %} Job roles list {% endblock %}
+
+
+{% block content %}
+<div>
+    <div class="container">
+
+    <h3>Job Role List</h3>
```

Comment:
in theory it should be one tab to right, because h3 and all other sections are nested within the div